### PR TITLE
added forgot password && checkbox

### PR DESCRIPTION
**Pull request by anjali1102**
CHECK PREVIEW 👉👉 [HERE](https://hogwarts-supplies-git-product-page-and-authen-d8c44f-anjali1102.vercel.app/pages/login.html)

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]

File: pages/forgot.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    <link rel="stylesheet" href="../style/forgot.css" />
+    <title>Forgot Password?</title>
+  </head>
+  <body>
+    <nav class="navbar_container">
+      <div class="navbar_main">
+        <a href="/"
+          ><img src="../assets/Hogwarts Supplies.png" class="brand-logo" alt=""
+        /></a>
```
Comment (VishalPatil18) [inline pages/forgot.html:19]: Code indentation could be improved.
e.g.
```HTML
<a href="/" >
  <img src="../assets/Hogwarts Supplies.png" class="brand-logo" alt="logo" />
</a>
```

File: pages/forgot.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    />
+
+    <link rel="stylesheet" href="../style/forgot.css" />
+    <title>Forgot Password?</title>
+  </head>
+  <body>
+    <nav class="navbar_container">
+      <div class="navbar_main">
+        <a href="/"
+          ><img src="../assets/Hogwarts Supplies.png" class="brand-logo" alt=""
```
Comment (VishalPatil18) [inline pages/forgot.html:18]: You could give value to the `alt` as that helps for accessibility

File: pages/forgot.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            <input
+              type="text"
+              class="search-box"
+              placeholder="search brand, product"
+            />
+            <button class="search-btn"><i class="fa fa-search"></i></button>
+          </div>
+          <a href="/pages/login.html"
+            ><img src="../assets/user.png" alt=""
+          /></a>
```
Comment (VishalPatil18) [inline pages/forgot.html:31]: Code indentation could be improved.

File: pages/forgot.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          <a href="#"><img src="../assets/cart.png" alt="" /></a>
```
Comment (VishalPatil18) [inline pages/forgot.html:32]: You could give value to the `alt` as that helps for accessibility

File: pages/login.html [hunk 5]
```diff
@@ -52,7 +52,15 @@ <h1>Login</h1>
           <span></span>
           <label>Password</label>
         </div>
-        <div class="pass">Forgot Password?</div>
+        <a href="/pages/forgot.html"
+          ><div class="pass">Forgot Password?</div></a
+        >
```
Comment (VishalPatil18) [inline pages/login.html:57]: Code indentation could be improved.

File: pages/forgot.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      <h1>Forgot Password? 😔</h1>
+      <form method="post">
+        <div class="txt_field">
+          <input type="text" required />
+          <span></span>
+          <label>Email Address</label>
+        </div>
+        <input type="submit" value="Send Email Link" />
+        <div class="signup_link">
+          Create Your Account | <a href="/pages/signup.html"><b>Signup</b></a>
```
Comment (VishalPatil18) [inline pages/forgot.html:48]: Great use of `<b></b>`. I wasn't aware of this. Got to learn something new.

**Review by VishalPatil18 — APPROVED** [review]
Just a few suggestions added around code indentation and `alt` attribute. 
The rest looks quite good. Great work @anjali1102 👍

File: style/login.css [hunk 7]
```diff
@@ -18,30 +19,30 @@ body {
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
-  width: 25rem;
+  width: 400px;
```
Comment (aniketxparihar) [inline style/login.css:22]: A suggestion:  relative units are more suitable than px values.

File: style/login.css [hunk 8]
```diff
@@ -59,24 +60,24 @@ form .txt_field {
 .txt_field span::before {
   content: "";
   position: absolute;
-  top: 2.5rem;
+  top: 40px;
   left: 0;
   width: 0%;
   height: 2px;
-  background: #2691d9;
+  background: #f12711;
```
Comment (aniketxparihar) [inline style/login.css:67]: You can use a variable for this

Comment (anjali1102) [reply style/login.css:67]: ok will do that later

**Pull request merged**